client/src/components/Landing.css

```diff
@@ -0,0 +1,46 @@
+
+#background {
+    
+    width: 100%;
+    height: 100%;
+    overflow: hidden;
+} 
+
+#background img {
+    position: absolute;
+    z-index: 1;
+    width: 100%;
+    height: 1000px;
+}
```

Comment:
Why not 100% height?

Reply:
I was testing how to crop images with CSS. I'll change it to 100%